animareflection/ui
# Skeleton Component

## Description

##### Task link: https://trello.com/c/VRIKrqme/119-skeleton-component

Added `Skeleton` component to universal core.

## Test Steps

1) Verify structure is sound for recipe, component, and stories.
2) Verify that `Skeleton` renders appropriately in storybook + downstream app(s).
